fritz.defaults.yaml

```diff
@@ -75,7 +77,7 @@ skyportal:
 
     kowalski:
       protocol: "http"
-      host: "kowalski_api_1"
+      host: "172.17.0.1"
```

Comment:
?

Reply:
This allows this to work on Linux

Reply:
Where does 172.17.0.1 come from?

Reply:
@dmitryduev

Reply:
i'm assuming this will get overwritten in the upcoming fritz tool refactor so approving with that expectation in mind.

Reply:
This is no longer in the diff - change has been removed